loading.go

```diff
@@ -88,6 +118,113 @@ func (rs *RiveScript) LoadDirectory(path string, extensions ...string) error {
 	return nil
 }
 
+/*
+LoadHttpFilesystem loads multiple RiveScript documents from a folder on disk.
+
+Parameters
+
+	path: Path to the directory on disk
+	extensions...: List of file extensions to filter on, default is
+	               '.rive' and '.rs'
+*/
+func (rs *RiveScript) LoadHttpFilesystem(httpfs http.FileSystem, path string, extensions ...string) error {
+	if len(extensions) == 0 {
+		extensions = []string{".rive", ".rs"}
+	}
+
+	files, err := vfspath.Glob(httpfs, fmt.Sprintf("%s/*", path))
```

Comment:
I'd rather not have this dependency be part of the rivescript-go core (see [RiveScript Goals and Scope](https://www.rivescript.com/contributing)). Currently the core of rivescript-go contains no third-party dependency outside the Go standard lib, and modules that do bring dependencies (sessions/redis, lang/javascript) must be chosen by the program author when they use RiveScript in their code.

LoadHttpFile() is okay to stay since it uses the standard net/http package.

It looks like you want this function to help with the bot program you're making, but this is probably better left as an implementation detail to your program and kept out of the rivescript-go core.